src/Services/ModerationService.cs

```diff
@@ -48,7 +48,8 @@ public Infraction AddTemporaryInfraction(TemporaryInfractionType type, ulong use
                 .WithDescription(reason)
                 .WithAdditionalInfo($"Duration: {duration.Humanize(7)}");
 
-            userData.Infractions.Add(infraction);
+            if (!skipPersistentInfractionRecord)
```

Comment:
What does this accomplish in a real-world scenario?

Reply:
This was added to make permanent infractions in the user data optional, so they don't have to be cleared. My personal reasoning was that a spam message is not enough to justify a persistent infraction.